packages/fields/src/types/Relationship/Implementation.js

```diff
@@ -232,9 +232,6 @@ export class Relationship extends Implementation {
 
   getGqlAuxTypes({ schemaName }) {
     const { refList } = this.tryResolveRefList();
-    if (!refList.access[schemaName].update) {
-      return [];
-    }
```

Comment:
We are not actually performing an update mutation on the target list, so this code can be removed.